runtime/doc/lsp.txt

```diff
@@ -1469,6 +1462,53 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
       • {client_id}  (integer)
 
 
+==============================================================================
+Lua module: vim.lsp.inlay_hint                                *lsp-inlay_hint*
+
+enable({bufnr}, {enable})                        *vim.lsp.inlay_hint.enable()*
+    Enable/disable/toggle inlay hints for a buffer
+
+    Parameters: ~
+      • {bufnr}   (integer|nil) Buffer handle, or 0 or nil for current
```

Comment:
Typo? Isn't `nil` for using all buffers?

Reply:
For `.get()` yes, but `.enable()`, no

Reply:
The inconsistency seems unnecessary, am I missing something?